lib/getContext.ts

```diff
@@ -2,10 +2,14 @@ const getContext = async (address: string) => {
     const BASE_URL = 'https://api.myco.wtf';
 
     const [tokensResponse, scoreResponse] = await Promise.all([
-        fetch(`${BASE_URL}/api/zora/tokens?creatorAddress=${address}`),
-        fetch(`${BASE_URL}/api/zora/score?address=${address}`)
+        fetch(`${BASE_URL}/api/zora/tokens?privyWalletAddress=${address}`),
+        fetch(`${BASE_URL}/api/zora/score?privyWalletAddress=${address}`)
```

Comment:
Why are you changing these params?
- Did you update each respective API to accept a different parameter name `privyWalletAddress`?